clients/fides-js/src/components/tcf/TcfPurposes.tsx

```diff
@@ -138,6 +149,7 @@ const TcfPurposes = ({
         onChange={(newEnabledIds) =>
           onChange({ newEnabledIds, modelType: "specialPurposes" })
         }
+        hideToggles
```

Comment:
I'm confused where this var comes from? It's not passed into the `TcfPurposes` const...

Reply:
oh this is a JSX shortcut, it's equivalent to 

```jsx
<Component hideToggles={true} />
```
but you can leave off the `={true}`. so `hideToggles` is the name of the prop on the component it's being passed into

Reply:
ohhh ok thanks for explaining!